coins/monero/src/wallet/send/mod.rs

```diff
@@ -235,9 +278,11 @@ pub struct SignableTransaction {
   protocol: Protocol,
   r_seed: Option<Zeroizing<[u8; 32]>>,
   inputs: Vec<SpendableOutput>,
+  decoys: Vec<Decoys>,
```

Comment:
Would it be better to do (SpendableOutput, Decoys) in a single vec? This is internal so I don't actually care, just noting.

Reply:
I like this suggestion. Done in the latest.